Assignment4/vcuende-180188/task06.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1L28eRU7IB0QsXz5d8h4hKsf-Bqc4exdt
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
This won't run in Python. You need to comment this line. The same applies for the other tasks

Reply:
Fixed